packages/expo-image/ios/ImageView.swift

```diff
@@ -416,7 +418,10 @@ public final class ImageView: ExpoView {
   }
 
   // MARK: - Live Text Interaction
-
+  #if os(tvOS)
+  private func analyzeImage() {}
+  var enableLiveTextInteraction = false
+  #else
```

Comment:
Do we need if/else here and adding these stubs? I'm more in favor of this 👇 and add appropriate ifs in places where we call `analyzeImage` and `enableLiveTextInteraction`. 
```suggestion

  #if !os(tvOS)
```

Reply:
Fixed